app/renderer/components/Session/AttachToSession.js

```diff
@@ -24,16 +24,9 @@ function formatCapsBrowserstack (caps) {
   return importantCaps.join(', ').trim();
 }
 
-function formatCapsLambdaTest (session) {
-  let caps;
-  if (session.capabilities) {
-    caps = session.capabilities;
-  } else if (session.desired) {
-    caps = session.desired;
-  } else {
-    caps = session;
-  }
-  let importantCaps = [caps.deviceName, caps.platformName, caps.platformVersion];
+function formatCapsLambdaTest (caps) {
+  let deviceName = caps.desired ? caps.desired.deviceName : caps.deviceName;
+  let importantCaps = [deviceName, caps.platformName, caps.platformVersion];
```

Comment:
```suggestion
  const importantCaps = [deviceName, caps.platformName, caps.platformVersion];
```

Reply:
When we were sending w3c as true, response was changing in session capabilities. Applied a fix, please have a look.

Reply:
@KazuCocoa

Reply:
Please fix the lint errors and let/const things i already commented

Reply:
Done @KazuCocoa

Reply:
Please check now @KazuCocoa

Reply:
Hopefully I'll take a look tonight